internal/openshift/controller.go

```diff
@@ -218,11 +218,11 @@ func (c *controllerImpl) userIdlerForNamespace(namespace string) *idler.UserIdle
 }
 
 // isActive returns true if build phase suggests a build is active, false otherwise.
-func (c *controllerImpl) isActive(b *model.Build) bool {
-	return model.Phases[b.Status.Phase] == 1
+func isActive(b model.Object) bool {
+	return model.Phases[b.Object.Status.Phase] == 1 && b.Type != "DELETED"
```

Comment:
Do you know what different values can 'b.Type` hold?

Reply:
https://github.com/kubernetes/apimachinery/blob/76721d167b7088b203cf9873c616b0cfec5c77bb/pkg/watch/watch.go#L44 

2 things
1. there are other values as well. I guess error should ignored as well or rather only look at `Added`, `Modfied` 

1. can we just the enum defined there instead of the "DELETED" magic word?